sources/packages/api/src-sql/EducationProgramsOfferings/Create-offered-type.sql

```diff
@@ -0,0 +1,9 @@
+-- Add the column offered_type for Offering with a dummy default value.
+ALTER TABLE sims.education_programs_offerings ADD 
+    COLUMN IF NOT EXISTS offered_type VARCHAR(50) NOT NULL DEFAULT 'fullTime';
+COMMENT ON COLUMN sims.education_programs_offerings.offered_type IS 'offered_type decides if offering is Full-Time and Part-Time';
```

Comment:
If the values are going to be Full-Time or Part-Time, can we add it as a enum type and assign sims.offerings_type here?

Reply:
mm, here values are coming from the formio and this default value is only for the migration purpose, it's a dummy value. I don't find the actual use of enum here. correct me if I misinterpreted it @guru-aot , let me know your thoughts @andrewsignori-aot

Reply:
I get it, it comes from formio. That's the real reason, I want to validate it along the db side too. I feel using these values we will create logics for the application going further. @andrewsignori-aot Thoughts please

Reply:
For offering I think an appropriate name would be intensity with values of Full Time Part Time. Thoughts?

Reply:
> I get it, it comes from formio. That's the real reason, I want to validate it along the db side too. I feel using these values we will create logics for the application going further. @andrewsignori-aot Thoughts please

Yes, we will probably be using this to apply some logic on API and/or workflow. I started a conversation with @abschwenker about when creating or not a Postgres enum, for instance. Let's review it in the upcoming meeting at 2:30 PM.